README.md

```diff
@@ -1,2 +1,185 @@
-# Portfolio
-This portfolio website is designed to show what I'm really capable of as a web developer
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="images/readme-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Portfolio README</b></h3>
+  <br/><hr /><hr /><br/><br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# Table of Contents 📋
+
+
+- [About the Project](#about-project)
+  - [Screenshot](#screenshot)
+  - [Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [Live Demo](#live-demo)
+- [Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [Author](#author)
+- [Future Features](#future-features)
+- [Contributing](#contributing)
+- [Show your support](#support)
+- [Acknowledgements](#acknowledgements)
+- [FAQ (OPTIONAL)](#faq)
+- [License](#license)
+
+    <br/>
+    <br/>
+<!-- PROJECT DESCRIPTION -->
+
+# <b>Portfolio</b> 📖<a name="about-project"></a>
+
+This is a portfolio website is designed to show what I'm really capable of as a web developer.<br/><hr /><br/><br/>
+## <b>Screenshot 📷</b><a name="screenshot"></a><hr />
+
+The screenshot below is a desktop version of the design.
+<div align="center">
+<br/>
+<img src="design/mobile version.jpeg" alt="app screenshot" width="auto"  height="auto" /> <br/><br/>
+<img src="design/menu.jpeg" alt="app screenshot" width="auto"  height="auto" />
+</div><br/>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<br/><br/>
+
+## <b>Built With 🛠</b> <a name="built-with"></a><hr />
+
+### <b>Tech Stack </b><a name="tech-stack"></a>
+
+This portfolio is build with:
+
+* Semantic <b>HTML5</b> markup to build raw structure of this web page
+* <b>CSS</b> custom properties to make the website visually attractive
+* <b>Node js</b> as a javascript runtime machine
+[git], a free and open source distributed version control system
+* <b>Linters</b> for coding convention and coding formating
+<br/><br/>
+
+<!-- Features -->
+
+### <b>Key Features</b> <a name="key-features"></a>
+<hr />
+
+> Coming soon...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<br/><br/>
+
+<!-- LIVE DEMO -->
+
+## <b>Live Demo 🚀</b> <a name="live-demo"></a>
+<hr />
+
+> Unfortunatly, there is not live demonstration avalaible right now 😭
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<br/><br/><br/><br/>
+<!-- GETTING STARTED -->
+
+## <b>Getting Started 💻</b><a name="getting-started"></a>
+<hr />
+
+> Coming soon...
+
+<br/><br/>
+
+### Prerequisites
+
+> Coming soon...
+
+### Setup
+
+> Coming soon...
+
+### Install
+
+> Coming soon...
+
+### Usage
+
+> Coming soon...
+
+### Run tests
+
+> Coming soon...
+
+<br/><br/>
+
+### <b>Deployment</b>
+
+> Coming soon...
+
```

Comment:
- [ ]  Please fill these areas with the information users need to install and run this project locally. If you feel there is nothing to write under a header, kindly remove them.